### PR TITLE
Refactored the help functionnality

### PR DESCRIPTION
**Pull request by DanStudent**

### CODE REVIEW

File: src/main/java/cli/ArithmeticRunner.java [hunk 1]
```diff
@@ -28,8 +28,9 @@ public Expression parse(String data) {
...
     @Override
     public String getHelp() {
-        return super.getHelp() +
-                "-------------------------------------------------- CALCULATOR HELP --------------------------------------------------\n\n" +
+        return super.getHelp() + "\n"  + loadHelpFile("ArithmeticHelper.txt");
+
+                /*"-------------------------------------------------- CALCULATOR HELP --------------------------------------------------\n\n" +
                 "This mode allows you to perform arithmetic operations.\n" +
                 "Expressions can be written in prefix, infix and postfix notation.\n\n" +
                 "In this mode, integers and rational numbers are supported: rational numbers are always noted as follows: 'r<integer>/<integer>'. For example: r5/2.\n\n" +
```
Comment (arpa42) [inline src/main/java/cli/ArithmeticRunner.java:36]: this comment should be removed

File: src/main/java/cli/BooleanRunner.java [hunk 2]
```diff
@@ -28,8 +28,8 @@ public Expression parse(String data) {
 
     @Override
     public String getHelp() {
-        return super.getHelp() +
-                "-------------------------------------------------- BOOLEAN HELP --------------------------------------------------\n\n"+
+        return super.getHelp() + "\n"  + loadHelpFile("BooleanHelper.txt");
+                /*"-------------------------------------------------- BOOLEAN HELP --------------------------------------------------\n\n"+
                 "This mode allows you to perform boolean operations.\n"+
                 "Expressions can be written in prefix, infix and postfix notation.\n\n"+
                 "In this mode, true is represented by the number 1 and false is represented by the number 0 \n\n"+
```
Comment (arpa42) [inline src/main/java/cli/BooleanRunner.java]: same

File: src/main/java/cli/ConvertorRunner.java [hunk 3]
```diff
@@ -9,8 +9,8 @@ public Expression parse(String data) {
 
     @Override
     public String getHelp() {
-        return super.getHelp() +
-                "--------------------------------------------- CONVERTOR HELP ------------------------------------------------\n\n" +
+        return super.getHelp() + "\n"  + loadHelpFile("ConvertorHelper.txt");
+                /*"--------------------------------------------- CONVERTOR HELP ------------------------------------------------\n\n" +
                 "This mode allows you to convert units.\n" +
                 "You just have to write 3 parameters 'inputUnit' 'outputUnit' 'amount'.\n\n" +
                 "Example:  'kg lb 100' will convert 100 kilograms to pounds\n" +
```
Comment (ArnaudPalgen) [inline src/main/java/cli/ConvertorRunner.java:16]: same

**Review by arpa42 — COMMENTED** [review]
comments shoud be removed

**Pull request merged**